ra_aid/llm.py

```
@@ -1,93 +1,198 @@
+        api_key=api_key,
+        base_url=base_url,
+        temperature=0 if is_expert else (temperature if temperature is not None else 1),
+        model=model_name,
+    )
+
+
+def create_openrouter_client(
+    model_name: str,
+    api_key: str,
+    temperature: Optional[float] = None,
+    is_expert: bool = False,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)